mm/kfence/kfence_core.c

```diff
@@ -259,14 +259,14 @@ static inline unsigned long kfence_index_to_addr(int index)
 	return kfence_obj_to_addr(obj, index);
 }
 
-void *guarded_alloc(struct kmem_cache *cache, gfp_t gfp)
+void *guarded_alloc(struct kmem_cache *cache, size_t override_size, gfp_t gfp)
```

Comment:
Sorry, haven't noticed before: I think since these are global names, they will require a kfence_ prefix.

Reply:
Right, I just forgot to rename it.